runtime/src/lib.rs

```diff
@@ -1635,13 +1623,12 @@ impl_runtime_apis! {
 			// Trying to add benchmarks directly to the Session Pallet caused cyclic dependency
 			// issues. To get around that, we separated the Session benchmarks into its own crate,
 			// which is why we need these two lines below.
-			// use pallet_session_benchmarking::Pallet as SessionBench;
-			// use pallet_offences_benchmarking::Pallet as OffencesBench;
-			use frame_system_benchmarking::Pallet as SystemBench;
+/*			 use pallet_session_benchmarking::Pallet as SessionBench;
+			 use pallet_offences_benchmarking::Pallet as OffencesBench;
 
-			// impl pallet_session_benchmarking::Config for Runtime {}
-			// impl pallet_offences_benchmarking::Config for Runtime {}
-			impl frame_system_benchmarking::Config for Runtime {}
+			 impl pallet_session_benchmarking::Config for Runtime {}
+			 impl pallet_offences_benchmarking::Config for Runtime {}
+*/			impl frame_system_benchmarking::Config for Runtime {}
```

Comment:
Same

Reply:
historic reasons. those are included from substrate's frame are benched by parity. @Gauthamastro can I clean up commented ones? we will not re-bench them.

Reply:
We don't need to re-bench crates by parity immediately.